db-scheduler/src/main/java/com/github/kagkarlsson/scheduler/jdbc/AutodetectJdbcCustomization.java

```diff
@@ -44,7 +44,7 @@ public AutodetectJdbcCustomization(DataSource dataSource) {
 
             if (databaseProductName.equals(MICROSOFT_SQL_SERVER)) {
                 LOG.info("Using MSSQL jdbc-overrides.");
-                detectedCustomization = new MssqlJdbcCustomization();
+                detectedCustomization = new MssqlJdbcCustomization(c.getMetaData().getDriverName());
```

Comment:
As mentioned below I think the jTDS should be a separate Customization, and hence no need to change standard `MssqlJdbcCustomization`

Is the last release of jTDS from 2013??

Reply:
Yes, the latest version was released in 2013